### ISSUE TITLE
Gradient Descent Animations

### ISSUE THREAD

**Issue by IljaAvadiev** [World4AI/World4AI#34]
The linear regression chapter on gradient descent should include animations regarding stochastic, batch and minibatch gradient descent. We represent the dataset as a series of blocks and indicate which blocks are used in an svg animation.

**Comment by IljaAvadiev** [comment]
These topics were moved to the section: "Data, Modules, Optimizers, Losses" in the "Neural Network" chapter.
